docs/release-notes/stable versions.md

```diff
@@ -0,0 +1,31 @@
+---
+id: stable-versions
+title: Mia-Platform stable versions
+sidebar_label: Stable versions
+---
+
+A **stable release** is a version that has been tested as thoroughly as possible and is as reliable as possible. It does not have all the new features of a beta release.
+
+Those versions labeled as _stable_ are particularly suitable for on-premise updates and installations.
+
+Typically, a new stable version is announced every quarter of the year. Therefore, __four stable versions__ will be announced over a-year period.
+
+For each stable version, a **maintenance window** will be granted for a three-months period approximately, pending the next stable version.
+
+Thus, once a [major/minor version](/info/version_policy.md) is labeled as stable, patches will be provided along the maintenance window in case of relevant detected bugs. Then, we recommend regularly upgrading to the latest patch of the stable release.  
+Once a new stable release is announced, it is suggested to upgrade to the new stable version so as to have guaranteed future support in case of malfunctions or relevant bugs detection.
```

Comment:
```suggestion
Thus, once a [Major/Minor version](/info/version_policy.md) is labeled as stable, patches will be provided throughout  the Maintenance Window in case of relevant detected bugs. Then, we recommend regularly upgrading to the latest patch of the stable release.  
Once a new stable release is announced, it is suggested to upgrade to the new stable version to ensure guaranteed future support in case of malfunctions or relevant bugs detection.
```